src/System.Private.CoreLib/shared/Interop/Windows/Kernel32/Interop.EventWaitHandle.cs

```diff
@@ -14,13 +15,13 @@ internal static partial class Kernel32
         internal const uint CREATE_EVENT_MANUAL_RESET = 0x1;
 
         [DllImport(Interop.Libraries.Kernel32, SetLastError = true)]
-        internal static extern bool SetEvent(SafeWaitHandle handle);
+        internal static extern bool SetEvent(SafeWaitHandle? handle);
```

Comment:
Just out of curiosity, since these ones live in kernel32, how did you conclude this ones can be nullable?

Reply:
Good catch. This is actually bogus; they can't accept null (the P/Invoke marshaling layer will throw argument null exceptions, or at least should.  I'll go back and figure out how I ended up annotating these and fix it.